tests/components/tf/modules/exploration/exploration_scheduling_test.py

```diff
@@ -0,0 +1,59 @@
+from mava.components.tf.modules.exploration.exploration_scheduling import (
+    PieceWiseLinearTimestepScheduler as PWLTS,
+)
+
+
+class TestExplorationScheduling:
+    """Testing the exploration schedule."""
+
+    @staticmethod
+    def create_piecewise_linear_exploration_schedule() -> PWLTS:
```

Comment:
This should probably be a fixture

Reply:
I agree with @sash-a , can we please make this a `pytest.fixture` or use a parametrizing test (https://docs.pytest.org/en/6.2.x/parametrize.html).

Reply:
done! :)